ixofoundation/ixo-webclient
# chore(bonds): integrate bonds react_ui project

## Scope

https://nonaredteam.atlassian.net/browse/IXO-42

## Work Done

- Integrated bonds_ui react project into this app
- Updated routes to handle nav to new module
- Updated redux to handle new module

## Steps to Test

- delete node_modules and run `npm install`
- run `npm run dev`
- navigate to `/bonds` and click around
- Note that there is no data for this account so buy and sell wont work

## Screenshots

<img width="1756" alt="Screenshot 2020-02-17 at 11 37 16" src="https://user-images.githubusercontent.com/20910282/74641407-e7331a00-5179-11ea-9768-a320857a470c.png">
